serverless/functions/sync-webhook.protected.js

```diff
@@ -24,6 +24,23 @@ exports.handler = async function (context, event, callback) {
       return callback(null, response);
     }
 
+    // Remove user from speakers map
+    try {
+      await streamSyncClient.syncMaps('speakers').syncMapItems(event.Identity).remove();
```

Comment:
It it every possible for this event to occur but the user is still connected to the Room?

Reply:
Sync has a 5 second timeout and rooms has a 30 second timeout I think. So if something goes wrong yeah the user could probably linger in the room a little longer than in sync. 

We could use the rooms REST API to remove the user immediately. I could go either way on this. It would add a little complexity and it's an edge case but it would make rooms and sync more consistent for the client. If closing web tabs happens a lot it may be worth it.

Reply:
I think it depends on if we think losing sync connectivity is a terminal state. I think this approach is fine